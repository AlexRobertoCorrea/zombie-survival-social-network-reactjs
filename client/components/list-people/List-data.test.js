import React from 'react';

import ListData from './List-data';

describe('<ListData />', () => {
  describe('Renders people data correctly', () => {
    const classes = {
      cell: 'test',
      button: 'test'
    };
    const people = [{
      name: '',
      age: 16,
      gender: 'M',
      infected: null
    }];
    const handleClickOpenInfectDialog = () => {};
    
    const listData = shallow(<ListData
      classes={classes}
      people={people}
      handleClickOpenInfectDialog={handleClickOpenInfectDialog}
    />);
    
    it('Ensure that listData works as expected', () => {
      expect(listData).toMatchSnapshot();
    });
  });
});
