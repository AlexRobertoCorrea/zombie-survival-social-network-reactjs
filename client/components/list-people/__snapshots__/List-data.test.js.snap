// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListData /> Renders people data correctly Ensure that listData works as expected 1`] = `
<ListData
  classes={
    Object {
      "button": "ListData-button-01 test",
      "cell": "ListData-cell-02 test",
    }
  }
  handleClickOpenInfectDialog={[Function]}
  people={
    Array [
      Object {
        "age": 16,
        "gender": "M",
        "infected": null,
        "name": "",
      },
    ]
  }
/>
`;
